engine/class_modules/sc_death_knight.cpp

```diff
@@ -5935,6 +6006,10 @@ struct obliterate_strike_t : public death_knight_melee_attack_t
     may_miss = false;
     weapon = w;
 
+    deaths_due_cleave_targets = as<int>(p -> spell.deaths_due -> effectN( 2 ).base_value()) + 
+                                  s_data -> effectN ( 1 ).chain_target() + 
```

Comment:
```suggestion
                                  data().effectN ( 1 ).chain_target() + 
```

The module usually uses data() instead of directly calling s_data.